src/ripple/app/ledger/impl/MetaView.cpp

```diff
@@ -218,11 +197,11 @@ MetaView::unchecked_insert(
         using namespace std;
         items_.emplace_hint(iter, piecewise_construct,
             forward_as_tuple(sle->key()),
-                forward_as_tuple(move(sle),
-                    taaCREATE, mSeq));
+                forward_as_tuple(taaCREATE,
+                    move(sle)));
```

Comment:
Thats exactly what I thought when I wrote it. But forget about that - the key is a member of the `STLedgerEntry` not the `shared_ptr`, so its all academic. Because its the `shared_ptr` thats moving, not the `SLE` it points to.